src/index.html

```diff
@@ -10,23 +10,59 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Search bar airbnb</title>
     <link
       rel="stylesheet"
-      href="style.css"
+      href="./style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search__bar big__bar"
+      method="post"
+      data-qa="big"
+    >
+      <label
+        for="biginput"
+        class="big"
+      >
+        <img
+          src="./images/Search.svg"
+          alt="search"
+          class="big__icon"
+        />
```

Comment:
Redundant, because you need to add icon with background property
<img width="507" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/64750958/a7f87587-2f8b-44b5-89f4-e5bd6f038535">
```suggestion
```